resources/letters/models/letter_base.yml

```diff
@@ -34,6 +35,7 @@ properties:
     oneOf:
       - $ref: "../../../shared/attributes/model_ids/adr_id.yml"
       - $ref: "../../../shared/models/address/inline_address.yml"
+      - $ref: "../../../shared/models/address/address_editable.yml"
```

Comment:
Why is this either inline address or address editable?

Reply:
Good catch. Feel into that encoding trap yet again. I needed to move `to` and `from` out of `base` and into `editable` and `generated_base`.

`to` and `from` in `generated_base` are now the full address model, which I rather thought they should be. All of the address model is now shared.